### PR TITLE
EPMDJ-5793: Remove "any" types from project

### PR DESCRIPTION
**Pull request by IliaSolovev**

### CODE REVIEW

File: src/pages/agent-full-page/coverage-plugin/coverage-details/coverage-cell/coverage-cell.tsx [hunk 1]
```diff
@@ -3,17 +3,13 @@ import { BEM, div } from '@redneckz/react-bem-helper';
...
 
-interface Props {
-  className?: string;
-  value: number;
-}
-
 const coverageCell = BEM(styles);
 
-export const CoverageCell = coverageCell(({ className, value }: Props) => (
+export const CoverageCell = coverageCell(({ className, value = 0 }: CellProps<number, unknown>) => (
```
Comment (RomanAchilov) [inline src/pages/agent-full-page/coverage-plugin/coverage-details/coverage-cell/coverage-cell.tsx:12]: why?

File: src/pages/agent-full-page/coverage-plugin/coverage-details/table/expanded-row-content.tsx [hunk 2]
```diff
@@ -3,30 +3,33 @@ import { BEM } from '@redneckz/react-bem-helper';
...
 import { SecondLevelExpandContent } from './second-level-expand-content';
 
 import styles from './table-row.module.scss';
 
-interface Props {
+interface Props<T, I> {
   className?: string;
-  item: any;
-  expandedColumns?: any[];
+  item: T;
```
Comment (RomanAchilov) [inline src/pages/agent-full-page/coverage-plugin/coverage-details/table/expanded-row-content.tsx:15]: it looks like this is a string

File: src/pages/agent-full-page/coverage-plugin/coverage-details/table/expanded-row-content.tsx [hunk 3]
```diff
@@ -3,30 +3,33 @@ import { BEM } from '@redneckz/react-bem-helper';
...
   className,
-  item = {},
+  item,
   expandedColumns = [],
   expandedRows = [],
   secondLevelExpand = [],
   classesTopicPrefix,
-}: Props) => {
+}: Props<T, I>) => {
   const { classes = [] } = useBuildVersion<Package>(`/${classesTopicPrefix}/coverage/packages/${item}`) || {};
```
Comment (RomanAchilov) [inline src/pages/agent-full-page/coverage-plugin/coverage-details/table/expanded-row-content.tsx:33]: `${item}` can be `[object object]`

File: src/pages/agent-full-page/coverage-plugin/coverage-details/table/table-types.ts [hunk 4]
```diff
@@ -1,6 +1,14 @@
 import * as React from 'react';
 
-export type Cell = React.ComponentType<any>;
+export interface CellProps<T, I> {
+  className?: string;
```
Comment (RomanAchilov) [inline src/pages/agent-full-page/coverage-plugin/coverage-details/table/table-types.ts:4]: remove

**Review by RomanAchilov — CHANGES_REQUESTED** [review]

File: src/components/wizard/wizard.tsx [hunk 5]
```diff
@@ -6,22 +6,28 @@ import {
...
   component: React.ComponentType<any>;
-  validate?: (fields: any) => { [fieldName: string]: string | string[] } | undefined;
+  validate?: FormValidator;
 }
 
 interface Props {
   className?: string;
   initialValues: Agent;
-  onSubmit: (val: any, onError: (message: string) => void) => any;
+  onSubmit: (val: Record<string, unknown>, onError: (message: string) => void) =>
```
Comment (RomanAchilov) [inline src/components/wizard/wizard.tsx:26]: only void

Comment (RomanAchilov) [reply src/components/wizard/wizard.tsx:26]: but you can leave it

**Review by RomanAchilov — CHANGES_REQUESTED** [review]

**Review by RomanDavlyatshin — APPROVED** [review]

File: src/pages/agent-full-page/coverage-plugin/coverage-details/table/expanded-row-content.tsx (hunk 3 above)
Comment (IliaSolovev) [reply src/pages/agent-full-page/coverage-plugin/coverage-details/table/expanded-row-content.tsx:33]: item is a string, I gave it a new type

**Pull request merged**